pkg/connector/tickets.go

```diff
@@ -325,7 +326,17 @@ func (j *Jira) CreateTicket(ctx context.Context, ticket *v2.Ticket, schema *v2.T
 				componentIDs = append(componentIDs, component.GetId())
 			}
 			ticketOptions = append(ticketOptions, WithComponents(componentIDs...))
+		case "issue_type":
+			issueType, err := sdkTicket.GetPickObjectValue(ticketFields[id])
+			if err != nil {
+				return nil, nil, err
+			}
+
+			if issueType.GetId() == "" {
+				return nil, nil, errors.New("error: unable to create ticket, issue type is required")
+			}
```

Comment:
`ValidateTicket` should take care of this

Reply:
removed https://github.com/ConductorOne/baton-jira/pull/13/commits/a80ecab3a2301b370b9f7eb56957e4aed1ee2f0c